src/virtual-select.js

```diff
@@ -681,12 +685,21 @@ export class VirtualSelect {
         return;
       }
 
+      if (this.markSearchResults) {
+        /** remove previous modifications to the label */
```

Comment:
Define a variable `markSearchResults` outside of the loop and use here instead of accessing directly from `this` object.